Assets/HoloToolkit/Utilities/Scripts/Editor/SaveLocalFileEditor.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using UnityEngine;
+using UnityEditor;
+
+namespace HoloToolkit.Unity
+{
+    [CustomPropertyDrawer(typeof(SaveLocalFileAttribute))]
+    public class SaveLocalFileEditor : PropertyDrawer
```

Comment:
Without some descriptive text, it's getting hard to determine whether these classes truly belong in Utilities or your examples folder.